app/Http/Controllers/PostController.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Post;
+
+class PostController extends Controller
+{
+    public function show($id)
+    {
+        if (!is_numeric($id)) {
```

Comment:
You are not validating this field on the other methods. It just looks ugly here :P (don't care about this comment)

Reply:
I am going to delete it. It is no longer necessary. It was because of the workshop 1.